Source/WebKit/WebProcess/GPU/graphics/WebGPU/RemoteGPUProxy.cpp

```diff
@@ -118,7 +118,7 @@ void RemoteGPUProxy::waitUntilInitialized()
     m_lost = true;
 }
 
-void RemoteGPUProxy::requestAdapter(const PAL::WebGPU::RequestAdapterOptions& options, CompletionHandler<void(RefPtr<PAL::WebGPU::Adapter>&&)>&& callback)
+void RemoteGPUProxy::requestAdapter(const PAL::WebGPU::RequestAdapterOptions& options, CompletionHandler<void(RefPtr<PAL::WebGPU::Adapter>&&)>&& callback, unsigned)
```

Comment:
This doesn't seem used?

This indicates a leaky abstraction.

Reply:
I will try to factor this away to avoid this issue